index.html

```diff
@@ -17,14 +17,15 @@
           </div>
 
           <nav class="desktop-nav">
+            <div class="x-nav"><img src="images/closeMark.png" alt="" /></div>
             <ul>
-              <li><a href="#">Portfolio</a></li>
-              <li><a href="#">About</a></li>
-              <li><a href="#">Contact</a></li>
+              <li><a href="#ancre1">Portfolio</a></li>
```

Comment:
- [ ] That is great implementation in each anchor tag has #id that is currently not navigating through anything because our sections are missing `id="ancreX" so let's make sure we set ids for our section in order for our navigation to work properly

Reply:
Thank you, I did that